app/workers/push_notifications/cleanup_worker.rb

```diff
@@ -0,0 +1,21 @@
+module PushNotifications
+  class CleanupWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform
+      redis = Redis.new(url: ENV["REDIS_RPUSH_URL"])
```

Comment:
In a lot of places, we're defaulting Redis URLs to `ENV["REDIS_URL"]`, should we do the same here?

```suggestion
      redis = Redis.new(url: ENV["REDIS_RPUSH_URL"] || ENV["REDIS_URL"])
```